galactic/distribution.yaml

```diff
@@ -2072,6 +2072,24 @@ repositories:
       url: https://github.com/eclipse/paho.mqtt.c.git
       version: master
     status: maintained
+  pal_statistics:
+    doc:
+      type: git
+      url: https://github.com/pal-robotics/pal_statistics.git
```

Comment:
all packages in the repo are using the same license, so maybe in the future you could add a top level license file instead of having one in each package subfolder (which would be easier to find).